html/changelogs/changelog20181.html

```diff
@@ -892,8 +892,8 @@ <h3 class='author'>Spookydonut updated:</h3>
 	</ul>
 	<h3 class='author'>Deanthelis updated:</h3>
 	<ul class='changes bgimages16'>
-		<li class='tweak'>Zombies can no longer be de-limbed by anything but brute force, i.e. shooting or hitting the limb. Predaliens can no longer take trophies from zombies.</li>
-		<li class='tweak'>Facehuggers cannot be placed directly on zombies by xenomorphs nor will they jump out of eggs, resin holes, or otherwise leap toward a zombie. Facehuggers treat zombies as though they do not exist.</li>
+		<li class='tweak'>Zombies can no longer be de-limbed by anything but brute force, i.e. shooting or hitting the limb. Predaliens can no longer take trophies from husks.</li>
+		<li class='tweak'>Facehuggers cannot be placed directly on husks by xenomorphs nor will they jump out of eggs, resin holes, or otherwise leap toward a husk. Facehuggers treat zombies as though they do not exist.</li>
```

Comment:
```suggestion
		<li class='tweak'>Facehuggers cannot be placed directly on husks by xenomorphs nor will they jump out of eggs, resin holes, or otherwise leap toward a husk. Facehuggers treat zombies as though they do not exist.</li>
```